Documentation/check-crd-compat-table.sh

```diff
@@ -49,6 +49,10 @@ get_rc_tags_for_minor(){
    git ls-remote --tags ${remote} v\* | awk '{ print $2 }' | grep "${minor_ver}" | grep -v '\^' | grep '\-' | sed 's+refs/tags/++' | sort -V
 }
 
+select_last_three() {
+    echo "${@}" | cut -d' ' -f2- -
```

Comment:
This works as long as we are sure to have 4 items in the list. How about:
```suggestion
    echo "${@}" | rev | cut -d' ' -f-3 | rev
```

Reply:
I tried this and it didn't work ;) Instead of reversing all of the characters in the stream, it reversed each individual argument then eliminated the wrong branch. I don't think it's worth getting this perfect.

Reply:
Agreed that it's not important.

But I'm getting curious now. It works for me, how can it possibly reverse each argument individually on your machine? :thinking: I would understand if the arguments were separated by newlines, but then the `cut` would not work as expected? (rev from util-linux 2.34 / GNU bash, version 5.0.17)

Reply:
Oh, it worked for me on the commandline, but not in the script ;)

I'm sure there's a way to get it to work, I just figured it's not a problem worth solving unless we decide to change the number of releases we are supporting at a given point in time.

Reply:
Actually, if we just don't call this "select_last_three" then it'd be generic to the count and actually do the right thing. I'll rename the function. Maybe "filter_out_oldest" is better.